src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/Configuration/ConfigurationKeys.cs

```diff
@@ -0,0 +1,48 @@
+namespace OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed.Configuration
+{
+    /// <summary>
+    /// Configuration keys
+    /// </summary>
+    public class ConfigurationKeys
+    {
+        /// <summary>
+        /// Configuration key for the application's default service name.
+        /// Used as the service name for top-level spans,
+        /// and used to determine service name of some child spans.
+        /// </summary>
+        public const string ServiceName = "OTEL_SERVICE";
+
+        /// <summary>
+        /// Configuration key for the application's version. Sets the "version" tag on every span.
+        /// </summary>
+        public const string ServiceVersion = "OTEL_VERSION";
+
+        /// <summary>
+        /// Configuration key for whether the tracer should be initialized by the profiler or not.
+        /// </summary>
+        public const string LoadTracerAtStartup = "OTEL_DOTNET_TRACER_LOAD_AT_STARTUP";
+
+        /// <summary>
+        /// Configuration key for the exporter to be used. The Tracer uses it to encode and
+        /// dispatch traces.
+        /// Default is <c>"Zipkin"</c>.
+        /// </summary>
+        public const string Exporter = "OTEL_EXPORTER";
+
+        /// <summary>
+        /// Configuration key for the Agent URL where the Tracer can send traces.
+        /// Default value is "http://localhost:8126".
+        /// </summary>
+        public const string AgentUri = "OTEL_TRACE_AGENT_URL";
```

Comment:
nit: this doesn't matter much for the PoC but I would rather use `OTEL_EXPORTER_ZIPKIN_ENDPOINT` as defined at https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#zipkin-exporter